tests/moe/test_unified_moe.py

```
@@ -2952,12 +3063,15 @@ def _make_bf16_from_logits_inputs(self, logits_dtype):
 
     @pytest.mark.parametrize("logits_dtype", [torch.float32, torch.bfloat16])
     def test_bf16_expert_weights_buffer_is_bf16(self, logits_dtype):
-        runner, _, moe_inputs, logits = self._make_bf16_from_logits_inputs(logits_dtype)
+        runner, inputs, moe_inputs, logits = self._make_bf16_from_logits_inputs(
```

Ruff (0.16.2)

[warning] Unpacked variable `runner` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)